stride/assetlist.json

```diff
@@ -175,6 +175,36 @@
         "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/stride/images/stluna.svg"
       }
     },
+    {
+      "denom_units": [
+        {
+          "denom": "stinj",
+          "exponent": 0
+        },
+        {
+          "denom": "stINJ",
+          "exponent": 18
+        }
+      ],
+      "base": "stinj",
+      "name": "stINJ",
+      "display": "stinj",
```

Comment:
Which means this should also be 'stINJ' to show that frontends should use the 18 exp unit.

Reply:
Ah good catch. Sorry about the back and forth. Done.